test/datasets/speechcommands_test.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from torchaudio.datasets import speechcommands
+
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    normalize_wav,
+    save_wav,
+)
+
+
+class TestSpeechCommands(TempDirMixin, TorchaudioTestCase):
+    backend = "default"
+
+    root_dir = None
+    samples = []
+
+    @classmethod
+    def setUp(cls):
+        cls.root_dir = cls.get_base_temp_dir()
+        dataset_dir = os.path.join(
+            cls.root_dir, speechcommands.FOLDER_IN_ARCHIVE, speechcommands.URL
+        )
+        os.makedirs(dataset_dir, exist_ok=True)
+        sample_rate = 16000  # 16kHz sample rate
+        for i in range(30):
+            label = "label_{:02d}".format(i)
+            path = os.path.join(dataset_dir, label)
+            os.makedirs(path, exist_ok=True)
+            for j in range(100):
+                # generate hash ID for speaker
+                speaker = "{:08x}".format(j)
+
+                for utterance in range(3):
+                    filename = f"{speaker}{speechcommands.HASH_DIVIDER}{utterance}.wav"
+                    file_path = os.path.join(path, filename)
+                    data = get_whitenoise(
+                        sample_rate=sample_rate,
+                        duration=0.01,
+                        n_channels=1,
+                        dtype="int16",
+                        seed=0,
+                    )
+                    save_wav(file_path, data, sample_rate)
+                    sample = (
+                        normalize_wav(data),
+                        sample_rate,
+                        label,
+                        speaker,
+                        utterance,
+                    )
+                    cls.samples.append(sample)
+
```

Comment:
Can you put back the `print` statement here? It does not affect test  behavior but it is helpful when debugging. `pytest -s ` will show the directory structure constructed.